staff/roberto-perez/e-commerce-template/index.html

```diff
@@ -0,0 +1,167 @@
+        </form>
+        <nav>
+            <button class="bars"><i class="fas fa-bars"></i></button>
+            <ul>
+                <li>
+                    <a href="#" title="Inicio">Inicio</a>
+                </li>
+                <li>
+                    <a href="#" title="Tienda">Tienda</a>
+                </li>
+                <li>
+                    <a href="#" title="Contacto">Contacto</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <article>
+                <div class="article-content">
+                    <a href="#">
+                        <img src="https://via.placeholder.com/450" />
+                    </a>
+                    <header>
+                        <h2>Producto nº 1</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur turpis quam, rutrum ac
+                            feugiat at.
+                        </p>
+                    </header>
+                </div>
+            </article>
+            <article>
+                <div class="article-content">
+                    <a href="#">
+                        <img src="https://via.placeholder.com/450" />
+                    </a>
+                    <header>
+                        <h2>Producto nº 1</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur turpis quam, rutrum ac
+                            feugiat at.
+                        </p>
+                    </header>
+                </div>
+            </article>
+            <article>
+                <div class="article-content">
+                    <a href="#">
+                        <img src="https://via.placeholder.com/450" />
+                    </a>
+                    <header>
+                        <h2>Producto nº 1</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur turpis quam, rutrum ac
+                            feugiat at.
+                        </p>
+                    </header>
+                </div>
+            </article>
+            <article>
+                <div class="article-content">
+                    <a href="#">
+                        <img src="https://via.placeholder.com/450" />
+                    </a>
+                    <header>
+                        <h2>Producto nº 1</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur turpis quam, rutrum ac
+                            feugiat at.
+                        </p>
+                    </header>
+                </div>
+            </article>
+            <article>
+                <div class="article-content">
+                    <a href="#">
+                        <img src="https://via.placeholder.com/450" />
```

Comment:
you could add an alt attribute to the img,

Reply:
Done!